### PR TITLE
EPMRPP-81646 || Implement plain table component

### PR DESCRIPTION
**Pull request by Bam6ycha**

### CODE REVIEW

File: app/src/componentLibrary/plainTable/plainTableRow/plainTableRow.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import classNames from 'classnames/bind';
+import Parser from 'html-react-parser';
+import { PlainTableCell } from '../plainTableCell';
+import { columnPropTypes } from '../propTypes';
+import styles from './plainTableRow.scss';
+
+const cx = classNames.bind(styles);
+
+export const PlainTableRow = ({ value, columns, actions }) => (
+  <div className={cx('row-wrapper')}>
```
Comment (AmsterGet) [inline app/src/componentLibrary/plainTable/plainTableRow/plainTableRow.jsx:27]: Let's name as component name `plain-table-row`.

File: app/src/componentLibrary/plainTable/plainTableRow/plainTableRow.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          id={id}
+          value={value}
+          title={title}
+          component={component}
+          align={align}
+          customProps={customProps}
+        />
+      ))}
+    </div>
+    {actions.length > 0 &&
```
Comment (AmsterGet) [inline app/src/componentLibrary/plainTable/plainTableRow/plainTableRow.jsx:41]: Why not to leave the iteration over empty array?)

File: app/src/componentLibrary/plainTable/plainTableCell/plainTableCell.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+export const PlainTableCell = ({ value, id, component, title, align, customProps }) => {
+  const CellComponent = component;
+
+  return (
+    <CellComponent
+      className={cx('table-cell', { [`align-${align}`]: align })}
+      title={title}
+      id={id}
+      customProps={customProps}
```
Comment (AmsterGet) [inline app/src/componentLibrary/plainTable/plainTableCell/plainTableCell.jsx:34]: Let's name it as `cellCustomProps` as it is intended only for cell.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/componentLibrary/plainTable/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+## **PlainTable**
+
+Has 930px width, min-height 80px
```
Comment (AmsterGet) [inline app/src/componentLibrary/plainTable/README.md:3]: Please update accordingly.
The height `80px` will be present in case of 1 row?

Comment (Bam6ycha) [reply app/src/componentLibrary/plainTable/README.md:3]: Oh, it is the height of one row without content.
I will fix the width and height values.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**